e2e_tests/docker_image/parsec-service-test-cross-compile.Dockerfile

```diff
@@ -0,0 +1,30 @@
+FROM ghcr.io/parallaxsecond/parsec-service-test-all
```

Comment:
Maybe we should "Copyright 2021 Contributors to the Parsec project." our Dockerfile as well, especially since they can be used by everyone once published?